tests/integration/buildcmd/build_integ_base.py

```diff
@@ -86,7 +87,7 @@ def get_command_list(
         if function_identifier:
             command_list += [function_identifier]
 
-        command_list += ["-t", self.template_path]
```

Comment:
can we override the template_path to be None in case of Terraform testing, and to depend on it in building the command list.